### PR TITLE
Proper Questions Formatting

### PR DESCRIPTION
**Pull request by Gosha04**
Changed questions and their corresponding db

### CODE REVIEW

File: web-app/public/questions.json
Comment (spencerau) [inline web-app/public/questions.json:1]: Does removing the submission ID break the program or anything? I wasn't involved with this part so I'm not too sure.

**Review by spencerau — COMMENTED** [review]
question about *.db file on questions.json

**Comment by spencerau** [comment]
Also can you change this to merge into 
Learn-Adapt-Build:dev-fall2024 from your main branch?

File: web-app/public/questions.json
Comment (Gosha04) [reply web-app/public/questions.json:1]: I'm not sure, I just kept it in.

**Pull request closed without merging**